File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int> res;
        while(n>0){
            int digit=n%10;
            res.push_back(digit);
            n/=10;
        }
        sort(res.begin(), res.end());
        int ans=res[res.size()-1]*res[res.size()-2];
        return ans;
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.java
class Solution {
    public int maxProduct(int n) {
        List<Integer> res = new ArrayList<>();
        while (n > 0) {
            int digit = n % 10;
            res.add(digit);
            n /= 10;
        }
        Collections.sort(res);
        int ans = res.get(res.size() - 1) * res.get(res.size() - 2);
        return ans;
    }
}